projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts

```diff
@@ -158,7 +160,7 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
           /** This function MUST be defined to enable dragging for the tree.
            *  Return false to cancel dragging of node.
            */
-          const draggable = _.get(this.config, 'mode') === 'Edit' ? true : false;
+          const draggable = _.get(this.config, 'mode') === 'edit' ? true : false;
```

Comment:
@vaibhavbhuva convert mode two lowercase and check

Reply:
Fixed.